qutebrowser/browser/commands.py

```diff
@@ -2171,6 +2171,7 @@ def fullscreen(self, leave=False):
 
         window = self._tabbed_browser.window()
         if window.isFullScreen():
-            window.showNormal()
+            window.setWindowState(window._state_before_fullscreen & ~Qt.WindowFullScreen)
         else:
+            window._state_before_fullscreen = window.windowState()
```

Comment:
Make this a public attribute (remove the leading `_`) if you want to access it from the outside.